### ISSUE TITLE
How to integrate with react-native-router-flux?

### ISSUE THREAD

**Issue by realtebo** [root-two/react-native-drawer#271]
Actually I've these scenes defined:

```
const scenes = Actions.create(
  <Scene key="root" hideNavBar open={false} >
    <Scene key="splashBucket">
      <Scene key="splashScene" component={SplashScreen} title="Splash Screen Title" hideNavBar />
    </Scene>
    <Scene key="authBucket">
      <Scene key="authScene" component={AuthForm} title="Accesso" hideNavBar={false} />
    </Scene>
    <Scene key="mainBucket" hideNavBar={false}>
      <Scene key="schedule" component={Schedule} title="Impegni"  />
      <Scene key="peopleList" component={PeopleList} title="Lista pesone"/>
    </Scene>
  </Scene>
);

const AppRoutes = () => (
  <Router 
    sceneStyle={{ paddingTop: 65 }} 
    scenes={scenes}
  />
);

export default AppRoutes;
```

I want your Drawer wrapped around the `mainBucket` scenes.
How to procede ?

**Comment by elieteyssedou** [comment]
I suggest you to take a look at https://github.com/infinitered/ignite, it is a CLI that generates React-Native apps with using both react-native-drawer and react-native-router-flux.

Hope it helps.

**Comment by AishwaryaSurana** [comment]
@realtebo are you able to integrate?

**Comment by realtebo** [comment]
@AishwaryaSurana No, I definetively switched to https://wix.github.io/react-native-navigation

**Comment by realtebo** [comment]
@AishwaryaSurana No, I definetively switched to React Native Navigation - truly native navigation for iOS and Android
  
|  
|   |  
React Native Navigation - truly native navigation for iOS and Android
 React Native Navigation - truly native navigation for iOS and Android  |  |

  |

 
 

    Il Martedì 5 Dicembre 2017 22:11, Eduardo Vazquez <notifications@github.com> ha scritto:
 

 @realtebo did you find an example in the documentation to implement it?—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.  

   
 
|  | Mail priva di virus. www.avast.com  |
